test/unit/component/parser/json/BpmnJsonParser.sequenceFlow.default.test.ts

```diff
@@ -82,11 +82,14 @@ describe('parse bpmn as json for default sequence flow', () => {
   });
 
   it(`should NOT convert, when an sequence flow (defined as default) is an attribute of 'process' and attached to a flow node where is NOT possible in BPMN Semantic`, () => {
```

Comment:
**thought:** I kept this test; however, given TypeScript's restriction that prevents accessing the `default` property on an object if it's not defined in the type, I'm not sure it's necessary.

Reply:
Remember that the BpmnJsonModel instance is created from an XML source in real live and no types check is done at runtime. So, at runtime, we can actually have to deal with such data.